README.md

```diff
@@ -53,9 +53,9 @@ All packages are placed in the [`packages/`](./packages) directory.
 
 | Package                                                                                                    | Version                                                                                                          | Description                                             |
 | ---------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------- |
```

Comment:
should probably add the new package to the list

Reply:
On it! We haven't pushed it to NPM yet, so its entry for NPM version column would not work (be broken)